red/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu
+
+##########################################################
+# General requirements for testing or common across many
+# implementations
+##########################################################
+
+RUN apt-get -y update
+
+# Required for running tests
+RUN apt-get -y install make python
+
+# Some typical implementation and test requirements
+RUN apt-get -y install curl libreadline-dev libedit-dev
+
+RUN mkdir -p /mal
+WORKDIR /mal
+
+##########################################################
+# Specific implementation requirements
+##########################################################
+
+RUN dpkg --add-architecture i386

```

Comment:
Does this mean the red system is only 32-bit at the moment? That's fine, just seems a bit odd that a modern programming language would not have 64-bit support.

Reply:
Yes, Red is only 32-bit for now, but the devs have said that it'll get 64-bit support by the end of the year.